examples/colourpicker-histogram.py

```diff
@@ -0,0 +1,48 @@
+from dash import Dash, dcc, html, Input, Output
+import plotly.graph_objects as go
```

Comment:
we can remove this package since we don't need to use it on line 40 of the code.